Dockerfile.dev

```diff
@@ -0,0 +1,9 @@
+FROM node:latest
+RUN mkdir /usr/src/app
+WORKDIR /usr/src/app
+ENV PATH /usr/src/app/node_modules:$PATH
+RUN npm install
+
+EXPOSE 8080
```

Comment:
Not necessary if we're leaning on `docker-compose` :) But if it's being built elsewhere and needs this then mah B 😄

Reply:
Good call.  This is left over from when I was trying to figure out why I couldn't access the server in the container from my local machine.  Will remove.